ui/src/message_popup/data_lists.jsx

```diff
@@ -105,7 +105,7 @@ export const allStudents = [
     gender: `female`,
```

Comment:
I think we can remove this file altogether now (this was probably my mistake with moving it to `virtual_school.js` but forgetting to remove this.  That way we don't have to update it in two places.  I'll do that separately now: https://github.com/mit-teaching-systems-lab/threeflows/pull/51